### DISCUSSION TITLE
ModuleNotFoundError: No module named 'gradio'

### DISCUSSION THREAD

**Discussion by AfaroX** [oobabooga/text-generation-webui#845] [General]
i get this for some reason
ModuleNotFoundError: No module named 'gradio'

**Comment by TomiTom1234** [comment]
Same issue here, I get this error message:

File "C:\AI\oobabooga-windows\text-generation-webui\server.py", line 14, in <module>
    import gradio as gr
ModuleNotFoundError: No module named 'gradio'

**Comment by pm4user444** [comment]
I spoke too soon, not fixed.

**Comment by pm4user444** [comment]
> this worked for me [#835 (comment)](https://github.com/oobabooga/text-generation-webui/issues/835#issuecomment-1499514571) #835
> run the micromamba-cmd.bat file
> pip install gradio
> then after that there were some other dependencies missing:
> pip install markdown peft protobuf
> then i was able to launch start-webui.bat properly

CAN CONFIRM, you need to run those commands in the CMD box  the micromaba-cmd bat makes.

**Reply by mockinbirdy** [reply]
Where is that file "micromamba-cmd.bat" cause I do not have it the text-generation-webui folder.

**Comment by mockinbirdy** [comment]
Same error. Still exists.
